README.adoc

```diff
@@ -0,0 +1,74 @@
+= Halkyon capability plugins architecture
+
+Part of what makes https://github.com/halkyonio/operator[Halkyon] interesting is the capability system. While the capability
```

Comment:
Can we add a link pointing where we cover `capability` within the halkyon operator and explaining what is a capability ?

Reply:
I'd argue that if you're reading about the capability plugin architecture, you already know what a capability is or you probably don't need to read this :)